contrib/spring_sleuth_v1x/src/main/java/io/opencensus/contrib/spring/sleuth/v1x/OpenCensusSleuthSpanContextHolder.java

```diff
@@ -52,10 +52,22 @@ static void setCurrentSpan(Span span) {
 
   // Remove all thread context relating to spans (useful for testing).
   // See close() for a better alternative in instrumetation
-  @SuppressWarnings("CheckReturnValue")
   static void removeCurrentSpan() {
-    CURRENT_SPAN.remove();
-    Context.current().withValue(ContextUtils.CONTEXT_SPAN_KEY, null).attach();
+    removeCurrentSpanInternal(null);
+  }
+
+  @SuppressWarnings("CheckReturnValue")
+  private static SpanContext removeCurrentSpanInternal(SpanContext toRestore) {
```

Comment:
`toRestore` needs to be marked as `/*Nullable*/`.

Reply:
Done.